test/unit/gie_self_tests.cpp

```diff
@@ -348,7 +348,9 @@ TEST(gie, info_functions) {
 
     /* proj_info()                                                            */
     /* this one is difficult to test, since the output changes with the setup */
+    putenv(const_cast<char *>("PROJ_IGNORE_USER_WRITABLE_DIRECTORY="));
```

Comment:
Again, ``PROJ_IGNORE_USER_WRITABLE_DIRECTORY`` has spread everywhere. Does this mean disable the cache or disable storing it in per-user locations like LOCALDATA?

Reply:
changed to PROJ_SKIP_READ_USER_WRITABLE_DIRECTORY